Adammathieson

adammathieson opened the pull request.

adammathieson committed:
added updatePlant fn with reducer and action

adammathieson committed:
Merge branch 'master' of https://github.com/team-watermyplants/frontend-water-my-plants into adammathieson

adammathieson committed:
fixed file erros in reducer/index.js

adammathieson committed:
added update fn with reducer and action

Merge branch 'master' of https://github.com/team-watermyplants/frontend-water-my-plants into adammathieson

cesarnml reviewed (approved):
app state logic is coming together nicely. good work! 🦄

cesarnml commented on src/reducers/index.js:
```diff
@@ -3,10 +3,14 @@ import { signUp } from "./signUp";
 import { combineReducers } from "redux";
 import { plantReducer } from "./plantReducer";
+import { deletePlantReducer } from ".deletePlantReducer";
+import { updateReducer } from ".updateReducer";
```
`import updateReducer from ".updateReducer"

You want to bring in a **default export** ... not a **named export** ... so no curly brackets needed.

cesarnml reviewed (changes requested):
Needs to fix a bug with import of updateReducer ... app crashing..

adammathieson committed:
fixed typo/bugs in deletePlantReducer, updateReducer, and reducer/index.js

adammathieson committed:
Fixed typos in deletePlantReducer, updateReducer, and reducer/index.js

Merge branch 'master' of https://github.com/team-watermyplants/frontend-water-my-plants into adammathieson
